easybuild/easyconfigs/p/ProteinMPNN/ProteinMPNN-1.0.1_20230627-foss-2022a.eb

```
@@ -0,0 +1,42 @@
+# Author: Lara Peeters (UGent)
+
+easyblock = 'Tarball'
+
+name = 'ProteinMPNN'
+version = '1.0.1_20230627'
+local_commit = '8907e6671bfbfc92303b5f79c4b5e6ce47cdef57'
```

@laraPPr I think it's more idiomatic to use:
```suggestion
local_commit = '8907e6671bfbfc92303b5f79c4b5e6ce47cdef57'
version suffix = '-CUDA-%(cudaver)s`
```
here and replace the later instances with `versionsuffix`.

You should then also add the version suffix to the filename.

---

@verdurin I've have updated the easyconfig